docs/Installation.md

```diff
@@ -520,10 +520,11 @@ Use the procedure for installation on [Debian](#2-installation-on-debian).
 
 ## 7. Post-installation
 
-### 7.1 Sanity check
+### 7.1 Smoke test of Zonemaster::Backend

```

Comment:
Do we really need to spell out Zonemaster::Backend in the heading? I think that is pretty much implied considering which document this is.